packs/databricks/config.yml

```diff
@@ -0,0 +1,39 @@
+
+# Name of the pack (required)
+name: databricks
+title: Databricks Integration
+description: >-
+  New Relic standalone integration polls the Apache Spark REST API in DataBricks cluster 
+  for metrics and pushes them into New Relic using Metrics API.
+summary: >-
+  New Relic standalone integration polls the Apache Spark REST API in DataBricks cluster 
+  for metrics and pushes them into New Relic using Metrics API.
+
+icon: icon.png
+logo: logo.png
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+website: https://databricks.com/
+
+keywords:
+  - nrlabs
+  - nrlabs-data
+  - apache spark
+  - spark
+  - databricks 
+
+# Authors of the pack (required)
+authors:
+  - New Relic Labs
+
+documentation:
+  - name: Databricks init script creator notebook
+    description: >-
+      Databricks notebook to create int script to be used during intialization of databricks cluster 
```

Comment:
small typos:
int -> init
intialization -> initialization
databricks -> DataBricks?

Reply:
fixed, thanks for catching these.